pegasus/sites.v3/code.org/public/css/page/lms-styles.scss

```diff
@@ -0,0 +1,41 @@
+section.top {
+  .pad {
```

Comment:
You can use the `.text-wrapper` selector here instead of adding this additional class.

Reply:
The 1rem didn't look as well-positioned against the image as the figma so I made a manual one for 1.5rem instead. Given that info- is it better to just take it out and use text-wrapper?

Reply:
You can keep these styles, but use the already-existing `.text-wrapper` class here instead of adding a new `.pad` class on the `.text-wrapper` element. Lmk if this still doesn't make sense!

Reply:
As in, just rename 'pad' to 'text-wrapper', correct?

Reply:
Yep! And then remove `.pad` in the haml file.

Reply:
Got it- done!